draft-ietf-quic-recovery.md

```diff
@@ -315,14 +315,16 @@ samples, and rttvar is the variation in the RTT samples, estimated using a
 mean variation.
 
 The calculation of smoothed_rtt uses path latency after adjusting RTT samples
-for ACK delays.  For packets sent in the ApplicationData packet number space,
-a peer limits any delay in sending an acknowledgement for an ack-eliciting
-packet to no greater than the value it advertised in the max_ack_delay transport
-parameter.  Consequently, when a peer reports an Ack Delay that is greater than
-its max_ack_delay, the delay is attributed to reasons out of the peer's control,
-such as scheduler latency at the peer or loss of previous ACK frames.  Any
-delays beyond the peer's max_ack_delay are therefore considered effectively
-part of path delay and incorporated into the smoothed_rtt estimate.
+for ACK delays. ACK delays are communicated in the ACK frame as an ack_delay
+field right shiften by the ack_delay_exponent transport parameter.  For packets
```

Comment:
```suggestion
field left shifted by the ack_delay_exponent transport parameter.  For packets
```
I know that shifting and endianness are often contended, but I think that this is a left shift.

Reply:
It seems confusing to me regardless.

Can we paraphrase the sentence to something like: _ACK delays are communicated using the ack_delay field of the ACK frame. The field contains ack delay in milliseconds, right shifted by the ack_delay_exponent transport parameter_?

Reply:
That seems right, apart from the bit where it should say "left".  Maybe we can instead say multiplied by 2 to the power of the ack_delay_exponent transport parameter value.

Reply:
I think I'd rather not talk about shifting, since we describe it differently in transport, how about 'adjusted'?

Reply:
And I added a reference to 19.3 of transport, see what you think.